conf/restperf/default.yaml

```diff
@@ -9,27 +9,38 @@ schedule:
 
 objects:
   CIFSNode:          cifs_node.yaml
-  Fcp:               fcp.yaml
+  Disk:              disk.yaml
+  ExtCacheObj:       ext_cache_obj.yaml
   FCVI:              fcvi.yaml
+  FcpPort:           fcp.yaml
```

Comment:
object name FcpPort?

Reply:
kept it same as zapiperf.. but i agree , we should change it to just fcp